build.gradle

```diff
@@ -158,6 +165,23 @@ project(':iceberg-common') {
       include 'NOTICE'
     }
 
+    dependencies {
+      exclude (dependency('org.checkerframework:checker-qual'))
+      exclude (dependency('org.codehaus.mojo:animal-sniffer-annotations'))
+      exclude (dependency('javax.annotation:javax.annotation-api'))
+      exclude (dependency('org.apache.orc:orc-core'))
+      exclude (dependency('org.jetbrains:annotations'))
+      exclude (dependency('org.apache.iceberg:iceberg-api'))
```

Comment:
`iceberg-common` doesn't depend on `iceberg-api` so I don't think this is needed?